src/modules/bill-runs-import/lib/queries.js

```diff
@@ -150,7 +158,15 @@ nbt2.charge_type,
 nbt2.net_amount,
 nbt2.legacy_id,
 nbt2.description,
-row_to_json(nbt) as metadata
+row_to_json(nbt) as metadata,
+nbt."SRCE_VALUE"::numeric,
+nbt."SEAS_VALUE"::numeric,
+nbt."LOSS_VALUE"::numeric,
+nbt."SUC_RATE"::numeric,
+case when nbt."ELEMENT_AGRMNTS" LIKE '%126' THEN REPLACE(nbt."ELEMENT_AGRMNT_VALS", 'x ', '')::numeric ELSE null END,
+case when nbt."ELEMENT_AGRMNTS" LIKE '%127' THEN REPLACE(nbt."ELEMENT_AGRMNT_VALS", 'x ', '')::numeric ELSE null END,
+nbt."EIUC_VALUE"::numeric,
+nbt."EIUC_SRCE_VALUE"::numeric
```

Comment:
Do we need:

For standard charges:
* calc_eiuc_factor to be null
* calc_eiuc_source_factor to be null

For compensation charges:
* calc_source_factor to be null
* calc_suc_factor to be null

Reply:
Copying the response from Slack:

> I see what you mean.
> I think the CM will return zeros rather than nulls and expects you to ignore them, so I kept that convention in place

Reply:
If we are keeping the CM logic then of zeros, I think we may need to move the selection of those 4 data points down in to the part of the query that splits a NALD transaction row into a separate standard and compensation charge